### ISSUE TITLE
Improvements [Unclosable thread]

### ISSUE THREAD

**Issue by noGreg** [noGreg/scribdStrip#1]
Hi, 

Leave here a comment about what you've done further this code, effects, etc., leave reference to your code if you start another project, if not just **make a pull request** ✌

**Comment by Cyberavater** [comment]
Does it work on studocu.com?

**Comment by noGreg** [comment]
@Cyberavater Yes! Download he repo since the extension is already removed from the Chrome Store

**Comment by Cyberavater** [comment]
> @Cyberavater Yes! Download he repo since the extension is already removed from the Chrome Store

Tried just now. Doesn't seem it works there.

**Comment by noGreg** [comment]
@Cyberavater Please make sure you're using the `dev` branch

**Comment by Cyberavater** [comment]
> @Cyberavater Please make sure you're using the `dev` branch

Yes, it works! thanks.

**Comment by Cyberavater** [comment]
Can you also add support for `everycircuit.com/app`?

**Comment by noGreg** [comment]
@Cyberavater I don't really see needs for. Please describe how is the user stocked with the popup.

**Comment by Cyberavater** [comment]
> @Cyberavater I don't really see needs for. Please describe how is the user stocked with the popup.

You'd need to try (run) out multiple (maybe around 10) circuits to get the popup.

![Screenshot (54)](https://user-images.githubusercontent.com/32187927/134132850-0a7bc3bf-2272-49f9-89d2-9949bc300757.png)

**Comment by GitHubRo80** [comment]
Working thx ! Is not possible when scroll, automatic un-blurr the content ?!

**Comment by noGreg** [comment]
> Working thx ! Is not possible when scroll, automatic un-blurr the content ?!
> I use chrome and dont have like you..the S up !

Actually it is possible. I will do it soon and let you guys know
